sdk/communication/azure-communication-chat/README.md

```diff
@@ -1,20 +1,23 @@
 # Azure Communication Chat Service client library for Android
-This package contains the Android SDK for Azure Communication Services for Chat.
-Read more about Azure Communication Services [here](https://docs.microsoft.com/azure/communication-services/overview).
+
+This package contains the Chat client library for Azure Communication Services.
+
+[Source code](https://github.com/Azure/azure-sdk-for-android/tree/master/sdk/communication/azure-communication-chat)
+| [API reference documentation](https://azure.github.io/azure-sdk-for-android/sdk/communication/azure-communication-chat/azure-communication-chat/index.html)
+| [Product documentation](https://docs.microsoft.com/azure/communication-services/overview)
 
 # Getting started
 
 ## Prerequisites
 
-* You must have an [Azure subscription](https://azure.microsoft.com/free/) to use this library.
-* An Azure Communication Resource, learn how to create one from [Create an Azure Communication Resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource).
+* You must have an [Azure subscription](https://azure.microsoft.com/free/) and a [Communication Services resource](https://docs.microsoft.com/azure/communication-services/quickstarts/create-communication-resource) to use this library.
```

Comment:
just curious, why are we removing the need for azure communication resource from prerequisite?

Reply:
never mind, I see it's just reformatted.

Reply:
It's just combined into one sentence to match the azure-communication-common readme.